demisto_sdk/commands/common/docker_helper.py

```diff
@@ -63,6 +93,28 @@ def init_global_docker_client(timeout: int = 60, log_prompt: str = ""):
         if ssh_client := os.getenv("DOCKER_SSH_CLIENT") is not None:
             logger.debug(f"{log_prompt} - Using ssh client setting: {ssh_client}")
         logger.debug(f"{log_prompt} - Using docker mounting: {CAN_MOUNT_FILES}")
+        try:
+            if IS_CONTENT_GITLAB_CI:
```

Comment:
Do we want an else statement here?

Reply:
No need. If we are running in CI pipeline this `if ` segment should handle all possible cases or fail if there is an issue initializing the docker client.
If we are not in CI pipeline, the default non-proxy logic will be used.